zrml/court/src/lib.rs

```diff
@@ -42,6 +45,8 @@ mod pallet {
     pub const NOT_RESOLVED: DispatchError = DispatchError::Other("Resolved outcome does not exist");
     pub const OUTCOME_MISMATCH: DispatchError =
         DispatchError::Other("Submitted outcome does not match market type");
+    pub const UNKNOWN_BLOCK: DispatchError = DispatchError::Other("Unknown block");
+    pub const UNKNOWN_MARKET_ID: DispatchError = DispatchError::Other("Unknown market id");
```

Comment:
Would be nice to have those in the error enum.

Reply:
Done